pyproject.toml

```diff
@@ -73,3 +73,6 @@ line_length = 79
 profile = "black"
 skip_gitignore = "True"
 verbose = "False"
+
+[tool.cibuildwheel]
+before-all = "uname -a"
```

Comment:
What's up with this? Is it a hangover from the CI problems?

Reply:
No it's been there a while.
TBH I took a quick look + concluded I didn't understand it, but perhaps I didn't need to.
I will dig a bit deeper + see if I can work out what it's for...

Reply:
OK I think this is some junk I mistakenly copied from another implementation.
I will remove it.

Reply:
See https://github.com/SciTools/cf-units/pull/460/commits/55ba22182a689ad76c7f7dde48a2fff278664ae8